1.0/docs/start/psk/creating-a-page.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: default
+type: start
+shortname: Start
+title: Creating a new page
+subtitle: Polymer Starter Kit
+---
+
+## Creating a new page
```

Comment:
Either in this doc or in the Set up guide, we should show the contents of PSK (that is, the folder structure) so folks know their way around.